.github/workflows/nightly_testing.yaml

```diff
@@ -50,7 +50,7 @@ jobs:
 
       - name: Run tests
         run: |
-          poetry run meson test -C builddir --suite=nightly-small --print-errorlogs --no-stdsplit
+          poetry run meson test -C builddir --suite=nightly-small --num-processes=1 --print-errorlogs --no-stdsplit
```

Comment:
Should probably just use the solution that the PR checks use: https://github.com/hse-project/hse/blob/master/.github/workflows/validate_build.yaml#L9